.gitignore

```diff
@@ -2,4 +2,6 @@
 .Rhistory
 .RData
 .Ruserdata
-/package_test
\ No newline at end of file
+/package_test
+db_R_tutorial.html
```

Comment:
Don't you want to keep the html-output?
I would put the whole folder "db_R_tutorial_files" in the .gitignore
Let's discuss on Friday what else to put in the .gitignore...

Reply:
I think for R beginner pdf file is better to understand. But, Jonas and Gavin have keep the .html file so finally I think it's better to keep and pdf file and html file. Sorry for this bad commit.